### PR TITLE
Solution for issue 0055 jumpgame

### LINKED ISSUE #127: 0055 - Jump Game
## Description of the Problem
Given an array of non-negative integers, you are initially positioned at the first index of the array.

Each element in the array represents your maximum jump length at that position.

Determine if you are able to reach the last index.

**Example 1:**
```
Input: nums = [2,3,1,1,4]
Output: true
Explanation: Jump 1 step from index 0 to 1, then 3 steps to the last index.
```

**Example 2:**
```
Input: nums = [3,2,1,0,4]
Output: false
Explanation: You will always arrive at index 3 no matter what. Its maximum jump length is 0, which makes it impossible to reach the last index.
```

**Constraints:**
- `1 <= nums.length <= 3 * 10^4`
- `0 <= nums[i][j] <= 10^5`

## Code
```
class Solution:
    def canJump(self, nums: List[int]) -> bool:
```

## Link To The LeetCode Problem
[LeetCode](https://leetcode.com/problems/jump-game/)

### PR DESCRIPTION
**Pull request by Dilpreet047**
**IMPORTANT: Please do not create a Pull Request without an issue.**

### All Submissions:

* [x] I have read the CONTRIBUTING document.
* [x] My code is written in **Python3** and is ending with _.py_
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] I have checked that my submission does pass the test on LeetCode.com 
* [x] Does your filename follow the naming Conventions?
* [x] Have you linked your PR to an Issue?

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
Closes #127

### CODE REVIEW

**Comment by vJechsmayr** [comment]
Not assigned to issue - waiting for Assignee

File: LeetCode/0055_jump_game.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def canJump(self, nums: List[int]) -> bool:
+        n = len(nums)-1
+        for i in range(n-1,-1,-1):
+            if nums[i] + i>=n:
```
Comment (Vulwsztyn) [inline LeetCode/0055_jump_game.py:5]:
```suggestion
            if nums[i] + i >= n:
```

File: LeetCode/0055_jump_game.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def canJump(self, nums: List[int]) -> bool:
+        n = len(nums)-1
+        for i in range(n-1,-1,-1):
```
Comment (Vulwsztyn) [inline LeetCode/0055_jump_game.py:4]:
```suggestion
        for i in range(n-1, -1, -1):
```

File: LeetCode/0055_jump_game.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def canJump(self, nums: List[int]) -> bool:
+        n = len(nums)-1
```
Comment (Vulwsztyn) [inline LeetCode/0055_jump_game.py:3]:
```suggestion
        n = len(nums) - 1
```

File: LeetCode/0055_jump_game.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+        for i in range(n-1,-1,-1):
+            if nums[i] + i>=n:
+                n = i
+        return n == 0
+
```
Comment (Vulwsztyn) [inline LeetCode/0055_jump_game.py:8]:
```suggestion
```

**Comment by vJechsmayr** [comment]
@Dilpreet047 Not assigned to Issue - please open only PRs when you assigned to the issue! -closed

**Pull request closed without merging**